formats/developers/ome-units.txt

```diff
@@ -0,0 +1,251 @@
+the specification for a Units of Measurement API available from
+http://www.unitsofmeasurement.org.
+
+The classes have been constructed in a way that means they could in future be
+reworked as a wrapper for Units of Measurement API implementation developed
+at https://www.eclipse.org/uomo/ if future work on that project provides a
+more complete implementation.
+
+
+OME system of measurements
+^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Defines a system of measurements for:
+
+.. contents::
+  :depth: 1
+  :local:
+
+
+Angle
+-----
+
+Is based on the following unit:
+
+-  **rad** - radian (the SI unit of angle)
+
+And from this is derived the following units:
+
+-  **deg** - degree
+-  **gon** - gradian
+
+All angle units are freely convertible.
+
+
+Electric Potential (commonly called Voltage)
+--------------------------------------------
+
+Is based on the following unit:
+
+-  **V** - volt (the SI unit of electric potential)
+
+And from this is derived the following units:
+
+-  All SI range for volt from |10^24| yotta (**YV**) to |10^-24| yocto
+   (**yV**)
+
+All electric potential units are freely convertible.
+
+
+Frequency
+---------
+
+Is based on the following unit:
+
+-  **Hz** - hertz (the SI unit of frequency)
+
+And from this is derived the following units:
+
+-  All SI range for hertz from |10^24| yotta (**YHz**) to |10^-24| yocto
+   (**yHz**)
+
+All frequency units are freely convertible.
+
+
+Length
+------
+
+Is based on the following units:
+
+-  **m** - meter (the si unit of length)
+-  **in** - inch (the Imperial/US unit of length)
+-  **pixel** - a length measured in terms of the image pixel size
+-  **reference frame** - a length measured in terms of an arbitrary unit
+   based on the equipment used
+
+And from these are derived the following units:
+
+-  All SI range for meter from |10^24| yotta (**Ym**) to |10^-24| yocto
+   (**ym**)
+-  **Å** - ångströms
+-  **thou** - thou (or mil, 1/1000 of an inch)
+-  **li** - line (1/12 of an inch)
+-  **in** - inch
+-  **ft** - foot
+-  **yd** - yard
+-  **mi** - terrestrial mile
+-  **ua** - astronomical unit - *The official term is ua as the SI standard
+   assigned AU to absorbance unit.*
+-  **ly** - light year
+-  **pc** - parsec
+-  **pt** - typography point - *This unit should be limited to font sizes.*
+
+All length units are **NOT** freely convertible. It is not possible to convert
+to or from **pixel** and **reference frame**.
+
+Power
+-----
+
+Is based on the following unit:
+
+-  **W** - watt (the SI unit of power)
+
+And from this is derived the following units:
+
+-  All SI range for watt from |10^24| yotta (**YW**) to |10^-24| yocto
+   (**yW**)
+
+All power units are freely convertible.
+
+
+Pressure
+--------
+
+Is based on the following unit:
+
+-  **Pa** - pascal (the si unit of pressure)
+
+And from this is derived the following units:
+
+-  All SI range for pascal from |10^24| yotta (**YPa**) to |10^-24| yocto (**yPa**)
+-  **Mbar** - mega bar
```

Comment:
I did not include bar in the model. That might be a mistake. I think my reasoning was that it was not usually used on its own. Cannot find why I made that assumption.